pkg/kubelet/kubelet_node_status.go

```diff
@@ -165,7 +165,7 @@ func (kl *Kubelet) reconcileHugePageResource(initialNode, existingNode *v1.Node)
 		if !supportedHugePageResources.Has(string(resourceName)) {
 			delete(existingNode.Status.Capacity, resourceName)
 			delete(existingNode.Status.Allocatable, resourceName)
-			klog.Infof("Removing now unsupported huge page resource named: %s", resourceName)
+			klog.Infof("Removing unsupported huge page resource at present,named: %s", resourceName)
```

Comment:
I'm not clear on how this error message ("at present") is any more clear than the existing one. (I also feel like the existing message is a bit confusing). 

When is this message logged exactly and what is the info we want to convey?

Reply:
The meaning of this log is: deleting unsupported resources or deleting currently unsupported resources？
I think it is latter。
Do you think so？

Reply:
I agree, I think it's latter, looks like it's logged in the case when huge page resources are deleted if they are no longer supported (similar to what comment describes).

maybe something like is more clear?

```
Huge page size resource named %s is no longer supported; removing resource from node
```

Reply:
I agree.But another question,we don't know the resource is no longer supported or will support in the future.
Simply, ”Removing unsupported huge page resource named: %s.” ？

Reply:
that sounds good as well, but that's basically equivalent to the current log +- `"now"` :)

Reply:
You are right,I have fixed it, thanks very much.